### ISSUE TITLE
Need documentation for :show_sensitive_data_on_connection_error

### ISSUE THREAD

**Issue by ChuckIrvine** [elixir-ecto/postgrex#478]
I changed my database configuration and now I'm getting the error shown below:

```
** (RuntimeError) connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details
    (elixir) lib/keyword.ex:393: Keyword.fetch!/2
    (postgrex) lib/postgrex/protocol.ex:90: Postgrex.Protocol.connect/1
    (db_connection) lib/db_connection/connection.ex:66: DBConnection.Connection.connect/2
    (connection) lib/connection.ex:622: Connection.enter_connect/5
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Last message: nil
State: Postgrex.Protocol
```

I would like to set :show_sensitive_data_on_connection_error to true, but I don't know how. I tried the following setting, but it had no effect:

```
defmodule Izendor.Repo do
  use Ecto.Repo,
    otp_app: :izendor,
    adapter: Ecto.Adapters.Postgres,
    show_sensitive_data_on_connection_error: true
end
```

I saw the closed issue at https://github.com/elixir-ecto/db_connection/issues/185, but the information there didn't help me since Jose didn't specify where exactly to make the configuration change, at least not in enough detail to help me.

**Comment by josevalim** [comment]
Thanks, those have been documented in `ecto_sql` for each relevant adapter.

**Comment by ChuckIrvine** [comment]
I did a google search and wasn't able to find that documentation for some reason. I noticed that a couple more folks were having trouble finding that documentation also.

However, just after I posted this issue, and doing a little more searching I found https://elixirforum.com/t/ecto-create-a-repo-error/13820/13. The very last comment by @garthk helped me make the needed config change.

Thanks for your quick response!
